### PR TITLE
Remove Docker.io references from python 3.6 V3 Dockerfile.

### PR DESCRIPTION
**Pull request by pdthummar**
Releasing new Host versions.
[V3 Release](PASTE_V3_LINK_HERE)
[V2 Release](PASTE_V2_LINK_HERE)

---

### PR information

- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which has an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/development-docs/cleaning-up-commits.md).
- [ ] If applicable, the PR references the bug/issue that it fixes in the description.
- [ ] New Unit tests were added for the changes made and CI is passing.

### CODE REVIEW

File: host/3.0/buster/amd64/python/python36/python36-slim.Dockerfile [hunk 1]
```diff
@@ -77,7 +91,17 @@ RUN apt-get update && \
...
+    # mysql dependencies
+    apt-get install -y default-libmysqlclient-dev
+
+# Chrome headless dependencies
+# https://github.com/puppeteer/puppeteer/blob/main/docs/troubleshooting.md#chrome-headless-doesnt-launch-on-unix
+#RUN apt-get install -y xvfb gconf-service libasound2 libatk1.0-0 libc6 libcairo2 libcups2 \
+#    libdbus-1-3 libexpat1 libfontconfig1 libgbm1 libgcc1 libgconf-2-4 libgdk-pixbuf2.0-0 libglib2.0-0 \
+#    libgtk-3-0 libnspr4 libpango-1.0-0 libpangocairo-1.0-0 libstdc++6 libx11-6 libx11-xcb1 libxcb1 \
+#    libxcomposite1 libxcursor1 libxdamage1 libxext6 libxfixes3 libxi6 libxrandr2 libxrender1 libxss1 \
+#    libxtst6 ca-certificates fonts-liberation libappindicator1 libnss3 lsb-release xdg-utils wget
```
Comment (vrdmr) [inline host/3.0/buster/amd64/python/python36/python36-slim.Dockerfile:104]: Do we need to keep this if its commented?

File: host/3.0/buster/amd64/python/python36/python36-slim.Dockerfile [hunk 2]
```diff
@@ -58,7 +72,7 @@ RUN apt-get update && \
     curl https://packages.microsoft.com/keys/microsoft.asc | apt-key add - && \
     curl https://packages.microsoft.com/config/debian/9/prod.list > /etc/apt/sources.list.d/mssql-release.list && \
     # Needed for libss1.0.0 and in turn MS SQL
-    echo 'deb http://security.debian.org/debian-security jessie/updates main' >> /etc/apt/sources.list && \
+    # echo 'deb http://security.debian.org/debian-security jessie/updates main' >> /etc/apt/sources.list && \
```
Comment (vrdmr) [inline host/3.0/buster/amd64/python/python36/python36-slim.Dockerfile:75]: What is the impact of removing it - I would ask the MS-SQL team (at least our PM working with them) to inform the use of this package.

File: host/3.0/buster/amd64/python/python36/python36-slim.Dockerfile [hunk 3]
```diff
@@ -77,7 +91,17 @@ RUN apt-get update && \
     #  binutils
     apt-get install -y binutils && \
     #  OpenMP dependencies
-    apt-get install -y libgomp1
+    apt-get install -y libgomp1 && \
+    # mysql dependencies
+    apt-get install -y default-libmysqlclient-dev
```
Comment (vrdmr) [inline host/3.0/buster/amd64/python/python36/python36-slim.Dockerfile:96]: What does this do?

File: host/3.0/buster/amd64/python/python36/python36-slim.Dockerfile (hunk 1 above)
Comment (pdthummar) [reply host/3.0/buster/amd64/python/python36/python36-slim.Dockerfile:104]: updated as per the recommendation.

File: host/3.0/buster/amd64/python/python36/python36-slim.Dockerfile (hunk 3 above)
Comment (pdthummar) [reply host/3.0/buster/amd64/python/python36/python36-slim.Dockerfile:96]: this is mysql dependencies and it is already in dev.

File: host/3.0/buster/amd64/python/python36/python36-slim.Dockerfile (hunk 2 above)
Comment (pdthummar) [reply host/3.0/buster/amd64/python/python36/python36-slim.Dockerfile:75]: This line is commented in dev branch. No change from my side for this PR.

File: host/3.0/buster/amd64/python/python36/python36-slim.Dockerfile [hunk 4]
```diff
@@ -77,7 +91,17 @@ RUN apt-get update && \
...
+
+# Chrome headless dependencies
+# https://github.com/puppeteer/puppeteer/blob/main/docs/troubleshooting.md#chrome-headless-doesnt-launch-on-unix
+RUN apt-get install -y xvfb gconf-service libasound2 libatk1.0-0 libc6 libcairo2 libcups2 \
```
Comment (CooperLink) [inline host/3.0/buster/amd64/python/python36/python36-slim.Dockerfile]: Please do comment these out in the -slim image to match the current dev branch

Comment (pdthummar) [reply host/3.0/buster/amd64/python/python36/python36-slim.Dockerfile]: Updated as per dev branch.

**Pull request closed without merging**